src/korma/core.clj

```diff
@@ -541,17 +570,23 @@
    (var? ent) (recur @ent)
    :else      (throw (Exception. (str "Can't determine default fk for " ent)))))
 
+(defn- to-raw-key [ent k]
```

Comment:
new internal function to-raw-key returns a single item for normal keys or a sequence for composite keys.